test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -21,9 +21,26 @@
 
 namespace Microsoft.ML.Tests
 {
-    public class OnnxTransformTests : TestDataPipeBase
+
+    /// <summary>
+    /// A Fact attribute for Onnx unit tests. Onnxruntime only supported
+    /// on Windows, Linux (Ubuntu 16.04) and 64-bit platforms.
+    /// </summary>
+    public class OnnxFact : FactAttribute
     {
+        public OnnxFact()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ||
```

Comment:
The logic is correct. The skip is because MLNet is on CentOS image. The transform runs fine on Ubuntu (locally). We cannot test on Linux image in MLNet